### ISSUE TITLE
Docker for Mac won't start -- /usr/local/bin/docker doesn't exist

### ISSUE THREAD

**Issue by KarlLaw** [docker/for-mac#2037]
### Expected behavior
Docker to start

### Actual behavior
Docker doesn't start
Gives fatal error

### Information
I've tried multiple installs of both the Stable and Edge versions

  - Full output of the diagnostics from "Diagnose & Feedback" in the menu
Diagnostic ID: 7C2D924B-B930-40B8-9891-719126CDB9AE

Docker for Mac: version: 17.07.0-ce-mac26 (69202b202f497d4b6e627c3370781b9e4b51ec78)
macOS: version 10.12.6 (build: 16G29)
logs: /tmp/7C2D924B-B930-40B8-9891-719126CDB9AE/20170906-205817.tar.gz
failure: Docker.qcow2 missing: the VM has never been started
[OK]     db.git
[OK]     vmnetd
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux is not running
[OK]     virtualization VT-X
[OK]     app
[ERROR]  moby
         /Users/KarlLaw/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/console-ring does not exist
[OK]     system
[OK]     moby-syslog
[ERROR]  db
         /Users/KarlLaw/Library/Containers/com.docker.docker/Data/s40 does not exist
         Unexpected error (No such file or directory) connecting to /Users/KarlLaw/Library/Containers/com.docker.docker/Data/s40
         com.docker.db is not running
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  slirp
         Unexpected error ((Failure
  "Error connecting socket to 9p endpoint unix:/Users/KarlLaw/Library/Containers/com.docker.docker/Data/s51: Unix.Unix_error(Unix.ENOENT, \"connect\", \"\")")) connecting to /Users/KarlLaw/Library/Containers/com.docker.docker/Data/s51
         com.docker.vpnkit is not running
         slirp check failed with: Unix.Unix_error(Unix.ENOENT, "connect", "")
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         cannot find docker
         /var/run/docker.sock does not exist
         Unexpected error (No such file or directory) connecting to /var/run/docker.sock
         /Users/KarlLaw/Library/Containers/com.docker.docker/Data/s60 does not exist
         Unexpected error (No such file or directory) connecting to /Users/KarlLaw/Library/Containers/com.docker.docker/Data/s60
         docker ps failed
[OK]     menubar
[ERROR]  disk
         Docker.qcow2 missing: the VM has never been started

  - A reproducible case if this is a bug, Dockerfiles FTW
  - Page URL if this is a docs issue or the name of a man page

### Steps to reproduce the behavior

  1. ...Install Docker for Mac
  2. ...Start Docker for Mac

**Comment by EvelynLiu** [comment]
Hi, I think you might have the same problem with me.
Can you show me your error msg before you click the diagnose button?

**Comment by EvelynLiu** [comment]
I fixed the problem by removing the original bin file and creating new symbolic link
p.s. you need to make sure it is safe for you to remove your bin
`cd /usr/local
sudo rm bin
sudo mkdir bin
ln -s /Applications/Docker.app/Contents/Resource/bin/notary.bin`

**Comment by KarlLaw** [comment]
Thanks for the help. I will try this later tonight. The error message I get states:

Fatal Error
Symlink /usr/local/bin/docker doesn't exist

**Comment by KarlLaw** [comment]
Thanks for the help. I didn't have to remove the original bin file like you did, but your solution led me on the path to mine. As I was preparing to try your fix, I found that my bin file wasn't owned by me and wasn't accessible by others. So I changed it to be owned by my user:

sudo chown -R $(whoami) /usr/local/bin

and made it accessible to others:

sudo chmod -R u=rwX,go=rX /usr/local/bin

I'm not sure exactly which of these fixed it. But the next time I tried opening Docker, it worked like a charm.

By the way, the error message I got was:

Fatal Error
Symlink /usr/local/bin/docker doesn't exist
